bootstrapper.yaml

```diff
@@ -65,7 +65,7 @@ spec:
     spec:
       containers:
       - name: deploy
-        image: gcr.io/ml-pipeline/bootstrapper:0.1.4 #TODO-release: update the release tag for the next release
```

Comment:
maybe use hash for the bootstrapper image?

Reply:
this change is specifically for sample images. the bootstrapper is deprecating. this file should be removed in a separate PR